packages/common/client-utils/package.json

```diff
@@ -34,18 +84,24 @@
 	"types": "dist/indexNode.d.ts",
 	"scripts": {
 		"api": "fluid-build . --task api",
-		"api-extractor:commonjs": "api-extractor run --local",
-		"api-extractor:esnext": "api-extractor run --config ./api-extractor-esm.json",
+		"api-extractor:commonjs": "api-extractor run --config ./api-extractor-cjs.json",
+		"api-extractor:esnext": "api-extractor run --local",
 		"build": "fluid-build . --task build",
 		"build:commonjs": "fluid-build . --task commonjs",
 		"build:compile": "fluid-build . --task compile",
 		"build:docs": "fluid-build . --task api",
-		"build:esnext": "tsc-multi --config ../../../common/build/build-common/tsc-multi.esm.json",
+		"build:esnext": "tsc --project ./tsconfig.json",
 		"build:test": "concurrently npm:build:test:mocha npm:build:test:jest npm:build:test:types",
-		"build:test:jest": "tsc --project ./src/test/jest/tsconfig.json",
-		"build:test:mocha": "tsc --project ./src/test/mocha/tsconfig.json",
-		"build:test:types": "tsc --project ./src/test/types/tsconfig.json",
-		"check:are-the-types-wrong": "attw --pack",
+		"build:test:jest:cjs": "fluid-tsc commonjs --project ./src/test/jest/tsconfig.cjs.json",
+		"build:test:jest:esm": "tsc --project ./src/test/jest/tsconfig.json",
+		"build:test:mocha:cjs": "fluid-tsc commonjs --project ./src/test/mocha/tsconfig.cjs.json",
+		"build:test:mocha:esm": "tsc --project ./src/test/mocha/tsconfig.json",
+		"build:test:types:cjs": "fluid-tsc commonjs --project ./src/test/types/tsconfig.cjs.json",
+		"build:test:types:esm": "tsc --project ./src/test/types/tsconfig.json",
+		"build:test:jest": "concurrently npm:build:test:jest:esm npm:build:test:jest:cjs",
+		"build:test:mocha": "concurrently npm:build:test:mocha:esm npm:build:test:mocha:cjs",
+		"build:test:types": "concurrently npm:build:test:types:esm npm:build:test:types:cjs",
```

Comment:
This feels like it's getting out of hand.  Do we need to compile type tests for both CJS and ESM?

I'm sure what running 'jest' picks up...  but I bet it's not both.  I assume ESM is the one to favor?